index.html

```diff
@@ -88,10 +88,18 @@ <h3>
         </div>
       </div>
 
+      <h2>Popular Community Blob Servers</h2>
```

Comment:
Maybe remove "Popular" from here? It doesn't strike me as appropriate.
```suggestion
      <h2>Community Blob Servers</h2>
```